defaults/main.yml

```diff
@@ -9,6 +9,7 @@ os_auth_timeout: 60
 os_auth_allow_homeless: false
 os_auth_pam_passwdqc_enable: true
 os_auth_pam_passwdqc_options: 'min=disabled,disabled,16,12,8'
+os_auth_pam_pwquality_options: 'try_first_pass retry=3 type='
```

Comment:
Should we add comments that this is the configuration for RHEL 7 and `os_auth_pam_passwdqc_options` is for <= RHEL 6?